contributed_definitions/nyaml/NXxps.yaml

```diff
@@ -0,0 +1,563 @@
+          angle_beam_analyser(NX_NUMBER):
+            exists: recommended
+            unit: NX_ANGLE
+            doc: |
+              Angle between incoming beam and the center analyser axis.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@depends_on:
+              doc: |
+                This could either point to the beam coordinate system (.) or to another transformation
+                that transforms from the beam coordinate system  to the XPS base coordinate system at
+                /entry/instrument/xps_coordinate_system.
+    (NXsample):
+      depends_on:
+        exists: recommended
+        doc: |
+          Reference to the transformation describing the orientation of the sample
+          relative to the beam.
+      transformations(NXtransformations):
+        angle_beam_sample(NX_NUMBER):
+          exists: recommended
+          unit: NX_ANGLE
+          doc: |
+            Angle between incoming beam and the sample.
+          \@transformation_type:
+            enumeration: [rotation]
+          \@depends_on:
+            doc: |
+              This could either point to the beam coordinate system (.) or to another transformation
+              that transforms from the beam coordinate system  to the XPS base coordinate system at
+              /entry/instrument/xps_coordinate_system.
+    peak_fitting(NXprocess):
+      exists: recommended
+      doc:
+      - |
+        Peak model for XPS fitting.
+      - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 3.29
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.29
+      (NXfit):
+        doc: |
+          Description of one peak fit. This is meant to be used for a peak fit in
+          _one_ XPS region. As an example, this could be used to describe the
+          fitting of one measured C 1s spectrum.
+        label:
+        data(NXdata):
+          doc: |
+            Input data and results of the fit.
+          input_dependent(NX_NUMBER):
+            unit: NX_ANY
+            doc: |
+              Independent axis for this fit procedure.
+              
+              This could be a link to entry/data/energy.
+          input_independent(NX_NUMBER):
+            unit: NX_ANY
+            doc: |
+              Dependent axis for this fit procedure.
+              
+              This could be a link to entry/data/data.
+          envelope(NX_NUMBER):
+          residual(NX_NUMBER):
+            exists: recommended
+        peakPEAK(NXpeak):
+          label:
+          data(NXdata):
+            position(NX_NUMBER):
+              unit: NX_ENERGY
+              doc: |
+                This could be a link to entry/data/energy.
+            intensity(NX_NUMBER):
+              doc:
+              - |
+                Intensity/count values at each energy in the position field.
+                This could be a link to entry/data/data.
```

Comment:
Well, this is the fit function, not the actual fitted data, no?

Reply:
yes, I changed it